org.kde.kst.yml

```
@@ -0,0 +1,112 @@
+app-id: org.kde.kst
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: kst2
+rename-desktop-file: kst2.desktop
+finish-args:
+  - '--share=network'
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--filesystem=home'
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /lib/*.la
+  - /lib/*.a
+  - /mkspecs
+  - /share/aclocal
+  - /share/info
+  - /share/man
+modules:
+
+  - name: gsl
+    sources:
+      - type: archive
+        url: 'https://ftp.gnu.org/gnu/gsl/gsl-2.7.1.tar.gz'
+        sha256: dcb0fbd43048832b757ff9942691a8dd70026d5da0ff85601e52687f6deeb34b
+        x-checker-data:
+          type: anitya
+          project-id: 1267
+          stable-only: true
+          url-template: 'https://ftp.gnu.org/gnu/gsl/gsl-$version.tar.gz'
+
+  - name: cfitsio
+    sources:
+      - type: archive
+        url: >-
+          https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-4.1.0.tar.gz
+        sha256: b367c695d2831958e7166921c3b356d5dfa51b1ecee505b97416ba39d1b6c17a
+
+  - name: getdata
+    config-opts:
+      - '--disable-bindings'
+      - '--enable-cplusplus'
+    sources:
+      - type: archive
+        url: https://github.com/ketiltrout/getdata/releases/download/v0.11.0/getdata-0.11.0.tar.gz
+        sha256: 082c241e04483c7cfbae5ad3ab9710d570051a888fb3f240f994b1cd61c9c26b
+
+  - name: matio
+    sources:
+      - type: archive
+        url: 'https://github.com/tbeu/matio/archive/v1.5.21.tar.gz'
+        sha256: 2b09738fc765609794cdd4745649dcb2d7a528e9ef427c3b8e2a0d71d37d02f7
+
+  - name: hdf5
+    config-opts:
+      - '--enable-cxx'
+    sources:
+      - type: archive
+        url: https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.8/hdf5-1.8.22/src/hdf5-1.8.22.tar.gz
+        sha256: 8406d96d9355ef8961d2739fb8fd5474ad4cdf52f3cfac657733defd9709bfaa
+      - type: shell
+        commands:
+          - cp -p /usr/share/automake-*/config.{sub,guess} bin
+
+  - name: netcdf
+    sources:
+      - type: archive
+        url: 'https://github.com/Unidata/netcdf-c/archive/v4.8.1.tar.gz'
+        sha256: bc018cc30d5da402622bf76462480664c6668b55eb16ba205a0dfb8647161dd0
+        x-checker-data:
+          type: anitya
+          project-id: 10354
+          stable-only: true
+          url-template: 'https://github.com/Unidata/netcdf-c/archive/v$version.tar.gz'
+      - type: shell
+        commands:
+          - cp -p /usr/share/automake-*/config.{sub,guess} bin
```

They are not in `bin` in this tarball.
```suggestion
          - cp -p /usr/share/automake-*/config.{sub,guess} .
```

---

@hfiguiere Thx, updated again!